Library/Formula/squirrelhive.rb

```diff
@@ -0,0 +1,57 @@
+require 'formula'
+
+class SquirrelHive < Formula
+  homepage 'http://www.squirrelsql.org/'
+  url 'http://sourceforge.net/projects/squirrel-sql/files/1-stable/3.5.0-plainzip/squirrel-sql-3.5.0-standard.tar.gz'
+  version '3.5.0'
+  sha1 'da39a3ee5e6b4b0d3255bfef95601890afd80709'
+
+  bottle do
+    cellar :any
+    sha1 "da39a3ee5e6b4b0d3255bfef95601890afd80709" => :yosemite
+    sha1 "da39a3ee5e6b4b0d3255bfef95601890afd80709" => :mavericks
+    sha1 "da39a3ee5e6b4b0d3255bfef95601890afd80709" => :mountain_lion
+  end
+
+  depends_on 'hadoop'
+  depends_on 'hive'
+  depends_on 'zookeeper'
+
+  def install; <<-EOS.undent
```

Comment:
This heredoc isn't being written anywhere, and there are no install steps in this formula as written.